package/boot/uboot-sunxi/Makefile

```diff
@@ -398,4 +398,17 @@ endef
 define Package/u-boot/install/default
 endef
 
+$(eval $(call TestHostCommand,swig,Please install 'swig', \
+	swig -version))
+
+$(eval $(call TestHostCommand,python3-dev, \
+	Please install the Python3 headers. , \
+	echo 'int main(int argc, char **argv) { Py_Initialize(); return 0; }' | \
+		gcc -include Python.h -x c -o $(TMP_DIR)/a.out - \
+		$$$$$$$$(python3-config --cflags --ldflags --embed)))
+
+$(eval $(call TestHostCommand,python3-pkg-resources, \
+	Please install the Python3 pkg_resources module, \
+	$(STAGING_DIR_HOST)/bin/python3 -c 'import pkg_resources'))
```

Comment:
I prefer to have this as a host dependency like your did.
Do you want to add this dependency only when uboot-sunxi is build?

Does it really work? It tried this some time ago and never got it really working, but I did it similarly to the thing in target/linux/x86/Makefile which I am pretty sure does not work.

Reply:
> Do you want to add this dependency only when uboot-sunxi is build?

If possible, yes, I would add it to every uboot target that require binman build.

> Does it really work? It tried this some time ago and never got it really working, but I did it similarly to the thing in target/linux/x86/Makefile which I am pretty sure does not work.

Sort of.

>> I've noticed that these prereqs are not checked again after one successful test, but target prereqs (in x86 and uml) also show the same behavior.

The prereq check will run again after deleting and re-creating .config, so I think it's better than nothing.